libs/external-router/src/model/type-route-pair.ts

```diff
@@ -0,0 +1,13 @@
+import { RouteWithoutPath } from './route-without-path';
+import { DaffExternalRouterType } from './route-type';
+
+/**
+ * A model used to describe the relationship between a {@link DaffExternalRouterType}
+ * and its corresponding route.
```

Comment:
I think this should mention that it's dealing with the "corresponding external route", because I think the relationship between the `TypeRoutePair` and the `DaffResolvableRoute` can be a little confusing. Unless I'm misunderstanding its purpose

Reply:
54ba78c6 I tried to clarify it a bit more.